package.json

```diff
@@ -25,9 +25,9 @@
   "devDependencies": {
     "aws-sdk": "^2.920.0",
     "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
+    "babel-loader": "^9.1.3",
     "babel-preset-env": "^1.7.0",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^3.3.12"
+    "webpack": "^5.94.0",
+    "webpack-cli": "^5.1.4"
```

Comment:
When user will be installing this lib in their dev environment, it will ask for updating this as well. do we need this behaviour?